package.json

```diff
@@ -5,7 +5,8 @@
   "main": "src/index.js",
   "type": "module",
   "scripts": {
-    "test": "c8 mocha",
+    "test": "c8 mocha test/*.test.js",
+    "test:integration": "mocha it/*.test.js",
```

Comment:
Just curious if this actually _only_ runs the ITs? When I tried to do this in the stock extractor, I found that the `mocha` definition in [package.json](https://github.com/adobe/content-lake-extractors-shared/blob/5fed66ad1366c7054bede5f6a0d180fd17177855/package.json#L25-L29) was causing the arguments to be ignored. I had to add `--no-package` as an option to the IT script command to get it to work like I expected. Was wondering if that was specific to my environment, or a general thing.

Reply:
Yep, I just double checked (added throw new Error in a unit test) it only runs the IT's